manifests/t/TTKN/CAJViewer/7.3.141/TTKN.CAJViewer.installer.yaml

```diff
@@ -0,0 +1,37 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: TTKN.CAJViewer
+PackageVersion: 7.3.141
```

Comment:
```suggestion
PackageVersion: 7.3.1
```

Reply:
It seems that all the 7.3.1** version of installers have the same package version 7.3.1 in the registry entry.

Reply:
Once you add the `7.3.1` version package into the repository, you should always update the existing one if Control Panel shows `7.3.1` rather than `7.3.XXX`.

Reply:
Hopefully winget schema version 1.1.0 is around the corner which have a workaround for these developer issues